src/fides/api/main.py

```diff
@@ -380,6 +192,38 @@ def read_other_paths(request: Request) -> Response:
     return get_admin_index_as_response()
 
 
+@app.on_event("startup")
+async def setup_server() -> None:
+    """Run all of the required setup steps for the webserver.
+
+    **NOTE**: The order of operations here _is_ deliberate
+    and must be maintained.
+    """
+
+    log_startup()
+
+    await run_database_startup()
+
+    check_redis()
+
+    if not scheduler.running:
+        scheduler.start()
+
+    initiate_scheduled_batch_email_send()
```

Comment:
@pattisdr I fixed the order here, but I don't remember if we allow the server to run without a scheduler? If so, I can add another conditional check here and raise an error if the server isn't running when the email batch gets added.

Likewise we can just leave it as-is 🙂